Add recipe for polish-holidays

przemarbor opened the pull request:
### Brief summary of what the package does

This package adds Polish holidays to the Emacs calendar.

### Direct link to the package repository

https://github.com/przemarbor/polish-holidays

### Your association with the package

I am the author/maintainer of the package.

### Relevant communications with the upstream package maintainer

None

### Checklist

<!-- Please confirm by replacing `[]` with `[x]`: -->

- [X] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [X] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [X] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [X] My elisp byte-compiles cleanly
- [X] I've used `M-x checkdoc` to check the package's documentation strings
- [X] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

<!-- After submitting, please fix any problems the CI reports. -->

PS. After building/installing the package Emacs (version 30.1) informs me about: "polish-holidays.el:1:1: Warning: file has no ‘lexical-binding’ directive on its first line", although I've included the line!

riscy commented:
> PS. After building/installing the package Emacs (version 30.1) informs me

A lot is package into the first line of the file - your first line should look like:
```
;;; polish-holidays.el --- Polish holidays -*- lexical-binding: t -*-
```

Checkdoc might've told you about this, although it's a little opaque:
```
polish-holidays.el:0: The first line should be of the form: ";;; package --- Summary"
```

⸺ `polish-holidays.el` with package-lint 20250418.1425:
```
1 issue found:
1:47: warning: You should depend on (emacs "24.1") if you need lexical-binding.
```

⸺ `polish-holidays.el` with [melpazoid](https://github.com/riscy/melpazoid):
```
- `;;; Commentary` is much wider than 80 characters
```

⸺ Package and license:
- Please specify `:fetcher` before `:repo` in your recipe
- Add a LICENSE file to the repository
  See: https://github.com/licensee/licensee

przemarbor commented:
I added melpazoid workflow to the repository, modified the code and succeeded in 
[running tests without errors](https://github.com/przemarbor/polish-holidays/actions/runs/15100679102/job/42441124808), though still getting a warning: "1:47: warning: You should depend on (emacs "24.1") if you need lexical-binding.". Can it be ignored or should I take steps to remove it?

riscy commented:
You can fix it by updating some of your frontmatter, e.g.
```elisp
;; Author: marbor
;; URL: https://github.com/przemarbor/polish-holidays
;; Version: 1.0.0
;; Keywords: calendar
;; Package-Requires: ((emacs "24.1"))
```

przemarbor commented:
Thank you @riscy.  After including the line you suggested, melpazoid check [returns no warning messages](https://github.com/przemarbor/polish-holidays/actions/runs/15241275917/job/42861645327).

riscy commented:
Thanks again for this - looks good!
